lib/Element.js

```diff
@@ -366,21 +365,13 @@ module.exports = class Element extends Node {
     this.$elm.remove();
     parentElement?._emitter.emit("_insert");
   }
-  replaceChild(newChild, oldChild) {
-    if (!newChild) {
-      throw new DOMException("Failed to execute 'replaceChild' on 'Node': parameter 2 is not of type 'Node'.");
+  replaceChildren(newChildren) {
+    while (this.lastChild) {
+      this.removeChild(this.lastChild);
```

Comment:
Not sure if this is an issue but `removeChild` / `appendChild` methods emits `_insert` events. Might result in an active mutation observer triggering multiple times during the execution of this method, rather than once after. Is this expected behavior? Otherwise might be proper to use cheerio internally

Reply:
Aaah, no, I just moved an implementation that we had in flamingo into tallahassee so that I could remove our local override of the class. I am not super familiar with hos these things are done in tallahassee, but I will look over the implementation!